doc/classes/GPUParticlesAttractorBox3D.xml

```diff
@@ -1,10 +1,11 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="GPUParticlesAttractorBox3D" inherits="GPUParticlesAttractor3D" version="4.1" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
 	<brief_description>
-		Box-shaped 3D particle attractor affecting [GPUParticles3D] nodes.
+		A box-shaped region inside which particles from [GPUParticles3D] nodes are influenced by an attractor.
```

Comment:
Too verbose for "brief description", both in terms of length and in language, especially as the same is available right below

Reply:
My general rule was to stay under 120 characters - this is 100, so I think it's good. I'd agree if it's overexplaining it, but I don't think that's the case. The nodes are not impacted at all: they are particle emitters, and the way they emit particles is completely unchanged.

Reply:
Since the same wording is used below I don't think it's required or is appropriate for this part

Reply:
hmm, a bit nitpicky but "are influenced by an attractor" sounds like the attractor is a separate thing, but as far as I know the box itself is the attractor. Maybe:
> A box-shaped region inside which particles from [GPUParticles3D] nodes are influenced.

Reply:
Seems correct, give me some time.